### DISCUSSION TITLE
HX711 and Weight Reset after DeepSleep

### DISCUSSION THREAD

**Discussion by tsunfinger** [arendst/Tasmota#10305] [Q&A] [answered]
I playing with a setup to permanent check weight by HX711/Tasmotab and iobroker in backend. Because of missing power supply I working with DeepSleep for saving power from accu. When ESP8266 comes back/wakes up it reset actual weight to 0.
Is there any option to get the actual weight after wake up?

**Comment by lalo-uy** [comment, marked as answer]
No, the driver logic is that it set zero to the poweron reading from the
sensor.
Previous calibration is not saved on eeprom.
I wanted to do the same, but don’t know where to save the calibration value.

**Comment by sfromis** [comment]
I think (not tried it) that you can save calibration values using the Sensor34 command from a boot rule.

**Comment by lalo-uy** [comment]
Will try it tomorrow.

**Comment by lalo-uy** [comment]
The problem is not save cal value, but remember previus one and set it at
boot. Will try with mem1 on rules.

**Reply by tsunfinger** [reply]
@lalo-uy: have you had success saving the calibration values?

**Reply by darkdevil373** [reply]
Very intresting, Can you tell how to setup?
